### PR TITLE
Homework 1

### PR DESCRIPTION
**Pull request by rohan-singh1**
- CSS Selectors completed
- Palindrome code completed
- Comment changed to conform to airbnb style guide

### CODE REVIEW

**Review by gleason9113 — APPROVED** [review]
Code looks good!  Only feedback I might offer is that using regex to validate the input string would save you some lines.

File: hw1/02-palindrome.js [hunk 1]
```diff
@@ -1,3 +1,40 @@
...
+     */
+    let result = 0;
+    for (let i = enteredValue; Math.floor(i) > 0; i /= 10) {
+      i = Math.floor(i);
+      l_digit = i % 10;
+      l_digit = Math.floor(l_digit);
+      result = result * 10 + l_digit;
+    }
+
+    if (enteredValue == result) {
```
Comment (rfilgas) [inline hw1/02-palindrome.js]: Style guide expects ===

File: hw1/02-palindrome.js [hunk 2]
```diff
@@ -1,3 +1,40 @@
...
+  } else {
+    /**
+     * Code to reverse a number sourced from my previous assignment: 
+     * https://github.com/rohan-singh1/student-repo-fullstack/blob/main/hw2/02-reverse.js
+     */
+    let result = 0;
+    for (let i = enteredValue; Math.floor(i) > 0; i /= 10) {
+      i = Math.floor(i);
+      l_digit = i % 10;
+      l_digit = Math.floor(l_digit);
```
Comment (rfilgas) [inline hw1/02-palindrome.js]: style guide expects camel case lDigit vs l_digit

File: hw1/02-palindrome.js [hunk 3]
```diff
@@ -1,3 +1,40 @@
 const elem = document.querySelector('input');
 
 elem.addEventListener('input', handleInput);
```
Comment (rfilgas) [inline hw1/02-palindrome.js:3]: style guide expects handleInput to be used after the function it is referencing

Comment (rfilgas) [inline hw1/02-palindrome.js]: style guide expects handleInput to be used after the function it is referencing in an argument

**Review by rfilgas — APPROVED** [review]
There are some style guide issues to catch I mentioned in some inline comments. I would recommend setting up eslint with airbnb's style preferences to catch those.

Detecting a palindrome is a bit easier using strings and built in functions, but this is a preference. Using strings would also allow your code to cover decimal numbers like 1.1 more easily for example.

File: hw1/02-palindrome.js (hunk 1 above)
Comment (rohan-singh1) [reply hw1/02-palindrome.js]: Feedback received and addressed.

File: hw1/02-palindrome.js (hunk 2 above)
Comment (rohan-singh1) [reply hw1/02-palindrome.js]: Feedback received and addressed.

File: hw1/02-palindrome.js (hunk 3 above)
Comment (rohan-singh1) [reply hw1/02-palindrome.js]: Feedback received and addressed. I configured ES Lint with my VS Code as per your recommendation and it showed me the errors you pointed out here, which I have resolved. I'm resolving the conversations as all the feedback has been incorporated into the code.

I had actually been using Prettier and it was auto formatting my code and as I had the ES Lint plugin installed I was under the impression that it was also handling the linter errors, until you pointed out that  I had to set it up manually, but at least it's working correctly now. Thank you so much man; I appreciate it.

**Comment by rohan-singh1** [comment]
Thanks for reviewing my PR gentlemen!

**Pull request merged**